### PR TITLE
Semesters show up as "OrgUnitName (Id: OrgUnitId)"

### PR DESCRIPTION
**Pull request by Vitalii-Misechko**
[US117427](https://rally1.rallydev.com/#/detail/userstory/396954104020?fdp=true): [Engagement][Filters] Semester

JFYI @anhill-D2L @MykolaGalian @rohitvinnakota @hyehayes

### Functional Testing

- Unit tests
- Verified manually

### UX and docs - N/A

image

### CODE REVIEW

**Comment by johngwilkinson** [comment]
UX: demo to Kevin. This is not consistent with the ou filter design, which shows ids like this: "Some Org Unit (Id: 12345)". I don't have a strong opinion about which way to do it, but I would think they should match. An tiny advantage of not having the term "Id" is that the code won't have to localize it.

**Comment by bemailloux** [comment]
Maybe check with both Megan and Kevin? The AC is inconsistent with the design (in screenshot below), so they should resolve that conflict.

image

**Comment by Vitalii-Misechko** [comment]
> both should be  OrgUnitName (Id: OrgUnitID)
https://d2l.slack.com/archives/CMNGJ22G0/p1597077957420800

**Comment by Vitalii-Misechko** [comment]
Updated screenshot

File: locales/en.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ export default {
 
 	"components.semester-filter.name": "Semesters",
 	"components.semester-filter.loadMore": "Load More",
+	"components.semester-filter.semester-name": "{orgUnitName} (Id: {id})",
```
Comment (johngwilkinson) [inline locales/en.js:12]: No change needed, but it occurs to me that if this were `{orgUnitName} (Id: {orgUnitId})`, you could pass it `item` directly.

Comment (Vitalii-Misechko) [reply locales/en.js:12]: Hm...sounds convenient

**Pull request merged**